src/test/java/com/exasol/udfstartuptimeimprover/UdfDefinitionParserTest.java

```diff
@@ -0,0 +1,133 @@
+package com.exasol.udfstartuptimeimprover;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.*;
+import static org.junit.jupiter.api.Assertions.assertAll;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import org.junit.jupiter.api.Test;
+
+class UdfDefinitionParserTest {
+    private static final UdfDefinitionParser PARSER = new UdfDefinitionParser();
+
+    @Test
+    void testParse() {
+        final UdfDefinition result = PARSER
+                .parseUdfDefinition("CREATE JAVA SCALAR SCRIPT \"MY_UDF\" (...) RETURNS VARCHAR(50) UTF8 AS\n"
```

Comment:
Add a test with `CREATE OR REPLACE` and another test with lower case (if this is can happen)